.github/workflows/inductor_xpu_e2e_nightly.yml

```diff
@@ -13,41 +13,25 @@ on:
         description: Specify triton commit, use pytorch pined commit by default
       suite:
         required: true
-        type: choice
+        type: string
         default: 'huggingface'
-        options:
-          - huggingface
-          - timm_models
-          - torchbench
-        description: Dynamo benchmarks test suite, huggingface / timm_models / torchbench
+        description: Dynamo benchmarks test suite. huggingface,timm_models,torchbench
```

Comment:
Add "support multiple suites split with ," in the description, similar change for other parameters